tests/test_copy_novaseqx_flow_cell.py

```diff
@@ -0,0 +1,126 @@
+from pathlib import Path
+from typing import Set
+from cg.cli.demultiplex.copy_novaseqx_data import get_latest_analysis_directory
+
+from cg.cli.demultiplex.demux import (
+    copy_flow_cell_analysis_data,
+    is_ready_for_post_processing,
+)
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+def test_flow_cell_is_ready_for_post_processing(
+    novaseqx_flow_cell_dir: Path, demultiplexed_flow_cell_run_directory: Path
+):
+    # GIVEN a flow cell which is ready for post processing
+
+    # WHEN checking if the flow cell is ready for post processing
+    ready = is_ready_for_post_processing(
```

Comment:
```suggestion
    is_flow_cell_ready = is_ready_for_post_processing(
```
I think it is good to not have to read the comment to understand what is ready.